### DISCUSSION TITLE
Publish to custom Closed Testing tracks

### DISCUSSION THREAD

**Discussion by IncrediblePony** [fastlane/fastlane#28302] [Q&A]
```ruby 
platform :android do
  desc "Push a new STAGING build to Google Play Store | Closed testing - Internal (staging)"
  lane :internal do
    gradle(
      task: 'bundle',
      build_type: 'Release',
      project_dir: "android/"
    )
    upload_to_play_store(
      skip_upload_apk: true,
      track: 'alpha',
      track_promote_to: 'Closed testing - Internal (staging)', # ID: '4701133352977831124', # Closed testing - Internal (staging)
      track_promote_release_status: 'completed',
    )
  end
end
``` 
I have a lane in my Fastlane setup. I want to upload to Google Play console Closed Testing. I have four tracks to choose from: 
```plaintext 
# UNPAUSED TRACKS IN GOOGLE CONSOLE
 * Closed testing - Closed testing - External/beta 
 * Closed testing - External (production) 
 * Closed testing - Internal (staging) 
 * Closed testing - Alpha

# PAUSED TRACKS IN GOOGLE CONSOLE
 * Closed testing - Not in use Beta
 * Closed testing - Beta
``` 

I want to specifically to use the tracks: `Closed testing - Internal (staging)` and `Closed testing - External (production)` 

The code shown above is attempting to upload to the Closed testing - Internal (staging) track. But the current setup only hits the Closed testing - Alpha track. It doesn't respond to the ID provided nor the name of the track. Based on the fastlane docs: https://docs.fastlane.tools/actions/supply/ I cannot seem to figure out how to make a config which achieves my endgoal. Is it even possible to target custom named closed testing tracks?

**Comment by IncrediblePony** [comment]
I really hoped someone had an answer :(

**Reply by rushicbs** [reply]
did you figure out the issue?

**Reply by IncrediblePony** [reply]
Nope. It seems that Google does not allow for you to specify a target track via API. If you want to point a build to a specific track you must do it via Google Play Console.
